src/wallet/rpcwallet.cpp

```diff
@@ -772,7 +772,7 @@ UniValue getbalance(const JSONRPCRequest& request)
             "\nArguments:\n"
             "1. \"account\"        (string, optional) DEPRECATED. The selected account, or \"*\" for entire wallet. It may be the default account using \"\".\n"
             "2. minconf          (numeric, optional, default=1) Only include transactions confirmed at least this many times.\n"
-            "3. addlocked      (bool, optional, default=false) Whether to add  balance from transactions locked via InstantSend.\n"
+            "3. addlocked      (bool, optional, default=false) Whether to add the balance from transactions locked via InstantSend.\n"
             "4. include_watchonly (bool, optional, default=false) Also include balance in watch-only addresses (see 'importaddress')\n"
```

Comment:
Wouldn't you want to be consistent and have "Also include the balannce in watch-only addresses (see 'importaddress')\n"

Reply:
Perhaps, but that is not consistent looking across all the RPCs (e.g. [listreceivedbyaddress](https://github.com/dashpay/dash/blob/develop/src/wallet/rpcwallet.cpp#L1300)). I just noticed the double-space and corrected it. Making them all consistent would be a bigger task and possibly complicate future backports from Bitcoin. 
@UdjinM6 @PastaPastaPasta What do you guys think?

Reply:
I agree that this change is rather trivial and poses more of an issue at upstream than what the change warrants.

Reply:
Imo non dash specific rpcs should be keep in line with upstream unless there is a substancial reason, ie not trivial.